### ISSUE TITLE
Cannot chain .mustAsync method

### ISSUE THREAD

**Issue by calebschoepp** [AlexJPotter/fluentvalidation-ts#18]
Maybe I missed this in the docs somewhere but I can't seem to chain `.mustAsync` and I'm unsure if I'm doing something wrong or if it is just not possible. The following example does not work. I'll keep digging in the docs and code but it'd be great if someone could help me understand why this does not work. Thanks :smile: 

```typescript
import { AsyncValidator } from 'fluentvalidation-ts';

type MyType = {
    field: string;
};

class Test extends AsyncValidator<MyType> {
    constructor() {
        super();

        this.ruleFor('field')
            .mustAsync(async () => true)
            .mustAsync(async () => true);
    }
}
```

**Comment by AlexJPotter** [comment]
Hi there,

Thanks for reporting this - it does appear to be an issue with the typings.

I'll look to get a fix out ASAP - in the meantime you should be able to work around this with two `.ruleFor` statements:

```typescript
this.ruleFor('field').mustAsync(async () => true);
this.ruleFor('field').mustAsync(async () => true);
```

You could also pull out the predicates and wrap them into custom rules, which can be chained together using an array (see [Custom Rules](https://fluentvalidation-ts.alexpotter.dev/docs/guides/customrules) for further info and some examples).

**Comment by AlexJPotter** [comment]
Hi there - this issue should now be fixed in [v2.2.2](https://github.com/AlexJPotter/fluentvalidation-ts/releases/tag/v2.2.2), please let me know if you have any further problems.

Thanks again for raising this issue! 😃

**Comment by calebschoepp** [comment]
Working like a charm. Thank you so much!
